### DISCUSSION TITLE
Tech Skill set for VSK

### DISCUSSION THREAD

**Discussion by Sharanbasavagouda** [Sunbird-cQube/community#763] [General]
To Install and configure VSK, implement state programs on VSK, request to share required Tech skill sets 

Request is form State Karnataka

**Comment by RNarmatha** [comment]
<html>
<body>
<div style="color: rgb(34, 34, 34); font-family: Arial, Helvetica, sans-serif; font-size: small; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><font face="garamond, times new roman, serif">The following requirements to install the cQube:</font></div><div style="color: rgb(34, 34, 34); font-family: Arial, Helvetica, sans-serif; font-size: small; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><font face="garamond, times new roman, serif"><br></font><div><span id="m_-4302105318186021475gmail-docs-internal-guid-b7ef8ca3-7fff-00f6-c4b5-506758a88f8c"><p dir="ltr" style="line-height: 1.38; text-align: justify; margin-top: 0pt; margin-bottom: 0pt;"><font face="garamond, times new roman, serif"><span style="font-size: 11pt; color: rgb(0, 0, 0); background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">2 servers are required on the state’s end: 1 for VSK and another for nginx. There are some other </span><a href="https://cqube.sunbird.org/use/prerequisites-for-installation-process" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://cqube.sunbird.org/use/prerequisites-for-installation-process&amp;source=gmail&amp;ust=1678553459874000&amp;usg=AOvVaw38vHT7Tixqk7g58-TrS4Lz" style="color: rgb(17, 85, 204); text-decoration-line: none;"><span style="font-size: 11pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; text-decoration-line: underline; vertical-align: baseline; white-space: pre-wrap;">prerequisites</span></a><span style="font-size: 11pt; color: rgb(0, 0, 0); background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"> as well for the state to install cQube as mentioned below:</span></font></p><ol style="margin-top: 0px; margin-bottom: 0px;"><li dir="ltr" style="margin-left: 15px; list-style-type: decimal; font-size: 11pt; color: rgb(0, 0, 0); background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"><p dir="ltr" role="presentation" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"><span style="font-size: 11pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"><font face="garamond, times new roman, serif">ubuntu 18.04 (supported)</font></span></p></li><li dir="ltr" style="margin-left: 15px; list-style-type: decimal; font-size: 11pt; color: rgb(0, 0, 0); background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"><p dir="ltr" role="presentation" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"><span style="font-size: 11pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"><font face="garamond, times new roman, serif">32GB of System RAM (minimum requirement)</font></span></p></li><li dir="ltr" style="margin-left: 15px; list-style-type: decimal; font-size: 11pt; color: rgb(0, 0, 0); background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"><p dir="ltr" role="presentation" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"><span style="font-size: 11pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"><font face="garamond, times new roman, serif">8 core CPU (minimum requirement)</font></span></p></li><li dir="ltr" style="margin-left: 15px; list-style-type: decimal; font-size: 11pt; color: rgb(0, 0, 0); background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"><p dir="ltr" role="presentation" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"><span style="font-size: 11pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"><font face="garamond, times new roman, serif">Domain name (with SSL)</font></span></p></li><li dir="ltr" style="margin-left: 15px; list-style-type: decimal; font-size: 11pt; color: rgb(0, 0, 0); background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"><p dir="ltr" role="presentation" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"><span style="font-size: 11pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"><font face="garamond, times new roman, serif">1 TB Storage</font></span></p></li></ol><br><a href="https://cqube.sunbird.org/use/software-requirements" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://cqube.sunbird.org/use/software-requirements&amp;source=gmail&amp;ust=1678553459874000&amp;usg=AOvVaw2vTUh698g_wDRDDPGBLMou" style="color: rgb(17, 85, 204); text-decoration-line: none;"><span style="font-size: 11pt; font-family: Calibri, sans-serif; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; text-decoration-line: underline; vertical-align: baseline; white-space: pre-wrap;">Here</span></a><span style="font-size: 11pt; font-family: Calibri, sans-serif; color: rgb(0, 0, 0); background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"> is the link to detailed software requirements to install cQube.</span></span><br></div></div><div style="color: rgb(34, 34, 34); font-family: Arial, Helvetica, sans-serif; font-size: small; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><span><span style="font-size: 11pt; font-family: Calibri, sans-serif; color: rgb(0, 0, 0); background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"><br></span></span></div><div style="color: rgb(34, 34, 34); font-family: Arial, Helvetica, sans-serif; font-size: small; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><span><span style="font-size: 11pt; font-family: Calibri, sans-serif; color: rgb(0, 0, 0); background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">In addition certain skillsets are required </span></span></div><div style="color: rgb(34, 34, 34); font-family: Arial, Helvetica, sans-serif; font-size: small; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><span id="m_-4302105318186021475gmail-docs-internal-guid-17393d4c-7fff-c6fd-eb6e-a26005509e1e"><p dir="ltr" style="line-height: 1.38; text-align: justify; margin-top: 0pt; margin-bottom: 0pt;"><span style="font-size: 11pt; font-family: Calibri, sans-serif; color: rgb(0, 0, 0); background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">A data analyst and a DevOps engineer are required in the state with the following skills:</span></p><p dir="ltr" style="line-height: 1.38; text-align: justify; margin-top: 0pt; margin-bottom: 0pt;"><br></p><div dir="ltr" align="left" style="margin-left: 18pt;">

Type of Engineer | Role | Skills Required
-- | -- | --
Data Analyst | Prepare the data to be ingested into NVSK / VSK Instances | Excel
DevOps | Deploy & Manage NVSK / VSK Instances | Basic understanding of cloud (Azure), Ansible, Shell scripting

</div></span></div><br class="Apple-interchange-newline">
</body>
</html>

**Comment by Sharanbasavagouda** [comment]
It is great to hear from you, thank you for the information and I
specifically required the skill sets/JD of the technical resources for
supporting installation, configuration and maintaining  VSK, can that be
shared?


On Fri, Mar 10, 2023 at 10:23 PM RNarmatha ***@***.***> wrote:

> The following requirements to install the cQube:
>
> 2 servers are required on the state’s end: 1 for VSK and another for
> nginx. There are some other prerequisites
> <https://cqube.sunbird.org/use/prerequisites-for-installation-process> as
> well for the state to install cQube as mentioned below:
>
>    1.
>
>    ubuntu 18.04 (supported)
>    2.
>
>    32GB of System RAM (minimum requirement)
>    3.
>
>    8 core CPU (minimum requirement)
>    4.
>
>    Domain name (with SSL)
>    5.
>
>    1 TB Storage
>
>
> Here <https://cqube.sunbird.org/use/software-requirements> is the link to
> detailed software requirements to install cQube.
>
> In addition certain skillsets are required
>
> A data analyst and a DevOps engineer are required in the state with the
> following skills:
>
>
> Type of Engineer Role Skills Required
> Data Analyst Prepare the data to be ingested into NVSK / VSK Instances
> Excel
> DevOps Deploy & Manage NVSK / VSK Instances Basic understanding of cloud
> (Azure), Ansible, Shell scripting
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Sunbird-cQube/discussions/763#discussioncomment-5269932>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEU3W7XG2J7PAW43M3QLI5LW3NL7XANCNFSM6AAAAAAVTRYLWI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Regards
Sharan

**Comment by Sharanbasavagouda** [comment]
looping  ananya

On Mon, Mar 13, 2023 at 3:36 PM Sharan Gouda ***@***.***> wrote:

> It is great to hear from you, thank you for the information and I
> specifically required the skill sets/JD of the technical resources for
> supporting installation, configuration and maintaining  VSK, can that be
> shared?
>
>
> On Fri, Mar 10, 2023 at 10:23 PM RNarmatha ***@***.***>
> wrote:
>
>> The following requirements to install the cQube:
>>
>> 2 servers are required on the state’s end: 1 for VSK and another for
>> nginx. There are some other prerequisites
>> <https://cqube.sunbird.org/use/prerequisites-for-installation-process>
>> as well for the state to install cQube as mentioned below:
>>
>>    1.
>>
>>    ubuntu 18.04 (supported)
>>    2.
>>
>>    32GB of System RAM (minimum requirement)
>>    3.
>>
>>    8 core CPU (minimum requirement)
>>    4.
>>
>>    Domain name (with SSL)
>>    5.
>>
>>    1 TB Storage
>>
>>
>> Here <https://cqube.sunbird.org/use/software-requirements> is the link
>> to detailed software requirements to install cQube.
>>
>> In addition certain skillsets are required
>>
>> A data analyst and a DevOps engineer are required in the state with the
>> following skills:
>>
>>
>> Type of Engineer Role Skills Required
>> Data Analyst Prepare the data to be ingested into NVSK / VSK Instances
>> Excel
>> DevOps Deploy & Manage NVSK / VSK Instances Basic understanding of cloud
>> (Azure), Ansible, Shell scripting
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/orgs/Sunbird-cQube/discussions/763#discussioncomment-5269932>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AEU3W7XG2J7PAW43M3QLI5LW3NL7XANCNFSM6AAAAAAVTRYLWI>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***
>> com>
>>
>
>
> --
> Regards
> Sharan
>


-- 
Regards
Sharan

**Reply by ananya-jain1** [reply]
Hi @Sharanbasavagouda - Following skill set is required to install and maintain the VSK Starter Pack:

Type of Engineer | Role | Skills Required
-- | -- | --
Data Analyst | Prepare the data to be ingested into NVSK / VSK Instances | Excel
DevOps | Deploy & Manage NVSK / VSK Instances | Basic understanding of cloud (Azure), Ansible, Shell scripting

<br class="Apple-interchange-newline">

A new version of cQube is also being launched in this month, we are preparing detailed skill sets / JD of the technical resources for the upcoming version. I can share that with you once done from our end.

**Reply by ananya-jain1** [reply]
Hi @Sharanbasavagouda - you can find the detailed skills required to install and maintain the latest version of cQube for VSK [here](https://sunbird.gitbook.io/sunbird-cqube/installation-and-upgradation/getting-started).
